packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -607,6 +612,127 @@ test("it supports {{itemViewClass=}} with in format", function() {
 
 });
 
+test("it supports {{emptyView=}}", function() {
```

Comment:
@mmun hmm okay. I'm a bit confused as to why the [itemView](https://github.com/emberjs/ember.js/blob/d1d5578cbefda479851db99074ab959671351794/packages/ember-htmlbars/tests/helpers/each_test.js#L490-L509) test is passing then. Any thoughts?